### PR TITLE
Deploy unstripped xen

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend [hunk 1]
```diff
@@ -30,4 +30,8 @@ do_deploy_append_rcar () {
     if [ -f ${D}/boot/${FLASK_POLICY_FILE} ]; then
         ln -sfr ${DEPLOYDIR}/${FLASK_POLICY_FILE} ${DEPLOYDIR}/xenpolicy
     fi
+
+    if [ -d ${DEPLOYDIR} ]; then
```
Comment (andr2000) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend]: I believe DEPLOYDIR always exists? Do you need to check?

Comment (iartemenko) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend]: you are right, useless check...

**Review by andr2000 — APPROVED** [review]

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend [hunk 2]
```diff
@@ -30,4 +30,6 @@ do_deploy_append_rcar () {
     if [ -f ${D}/boot/${FLASK_POLICY_FILE} ]; then
         ln -sfr ${DEPLOYDIR}/${FLASK_POLICY_FILE} ${DEPLOYDIR}/xenpolicy
     fi
+
+    find ${S}/xen -iname "xen-syms" -exec tar -cJvf ${DEPLOYDIR}/xen-syms.tar.xz {} \;
```
Comment (lorc) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend]: xen-syms location is known. It is located at `${S}/xen/xen-syms`. No need to use `find` there.

Comment (iartemenko) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend]: it was just to ensure it is really there, let's assume it is...

**Comment by lorc** [comment]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**